ADBench/plot_graphs.py

```diff
@@ -23,8 +23,10 @@
 
```

Comment:
I have found something interesting related to lines 21-22. If user runs this script to get a reference (`python plot_graphs.py --help`, for example) then warning message about `--show` flag enabling will be printed. That is because in this case `do_plotly` and `do_save` are false, so, `do_show` will be true. I think 21-22 lines shoul be mooved in place after reference printing (to line 196). What do you think?

Reply:
Good point.  This PR helps a bit https://github.com/awf/ADBench/pull/108

But we should subsequently refactor the script so no executable code runs at the top level.